src/main/java/org/kuali/kra/award/lookup/AwardLookupableHelperServiceImpl.java

```diff
@@ -1,310 +0,0 @@
-/*

```

Comment:
I've removed this file as I've unnecessarily brought it over from Foundation for this fix.